ui/round/css/_layout.scss

```diff
@@ -31,10 +31,10 @@
     @extend %zen;
 
     grid-area: uchat;
+  }
 
-    #{$move-tag} {
-      display: none;
-    }
```

Comment:
Not sure what that way about? I don't think this makes sense here?

Reply:
a remain of an old cheat bot detection battle I think